library/src/main/kotlin/build/buf/connect/protocols/ErrorJSONModels.kt

```diff
@@ -15,18 +15,22 @@
 package build.buf.connect.protocols
 
 import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
 
+@JsonClass(generateAdapter = true)
```

Comment:
This annotation will tell the KSP to generate adapter classes (which have toJson/fromJson methods) to support these types.